db/controller.js

```diff
@@ -21,9 +18,20 @@ exports.getArticleById = (req, res, next) => {
     .catch(next);
 };
 
-exports.updateVotes = (req, res) => {
+exports.updateVotes = (req, res, next) => {
   console.log(req.body);
-  alterVotes(req.body).then((alteredArticle) => {
-    res.send(200).send({ alteredArticle }).catch(next);
-  });
+  //console.log(!req.body.inc_votes, "<<<<<<<<!req.body.inc_votes");
+
+  if (!req.body.inc_votes) {
```

Comment:
```suggestion
  if (!req.body.inc_votes) {
```
This includes all falsey values. Notably 0 is falsey so your api would send a message of `"invalid input type"` if you sent `inc_votes: 0`

Reply:
Altered code to:   
```if (!req.body.inc_votes && req.body.inc_votes !== 0) ```